### DISCUSSION TITLE
Allow to configure what is shown in the feed

### DISCUSSION THREAD

**Discussion by baptiste0928** [community/community#13156] [Other Feature Feedback, Questions, & Ideas]
I personally find this new feed useful, especially since activities of starred repos are shown. However, the "social" features are quite annoying, especially the following suggestions. GitHub doesn't need that in my opinion, and the algorithm should be designed to show us useful content rather than push us to consume content and lose productivity.

It would be nice to allow customizing as much as possible what is displayed in the feed, and how it is displayed. For example, choose between ranking with an algorithm or just chronologically, or allow choosing exactly what kind of elements to display and hide some people or repos.

**Comment by jamieshark** [comment]
Thank you for your feedback! We have also considered the option of having various user-generated feed tabs that would only include content that you've selectively filtered and can also sort how you'd like. For example, a feed that only shows releases from certain repositories and sorted chronologically (or any other number of combinations you could create yourself). Are there certain filters (other than people and repos like you've mentioned) or sorts that you'd prioritize being available?

**Reply by baptiste0928** [reply]
Custom feeds tabs is a great idea! :+1: 

I think that a filter that shows only the dependencies of our repos could be very useful to follow their evolution and participate in the issues. Also, allowing to highlight security warnings or issues that need help (with the help-wanted label) could be useful.

Unrelated, but is there any plan to support dependency graphs for Rust repos? Is there any way to track the progress of this feature (if planned)?

**Reply by TechStudent10** [reply]
I would like the ability to hide stars and releases from certain users. For example, a developer on GitHub that is always active.  would really appreciate being able to hide stuff like releases and stars from this user.

**Reply by jamieshark** [reply]
Unfortunately not sure about status or plans regarding Rust dependency graphs, but we will consider including dependencies or perhaps issues of a certain type as a possible feed inclusion in the future!

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
